src/handlers/autolabel.rs

```diff
@@ -108,6 +113,14 @@ pub(super) async fn parse_input(
                     });
                     state.data.new_pr_labels_applied = true;
                 }
+
+                // If a PR is converted to draft remove all the "new PR" labels
+                if cfg.new_pr && event.action == IssuesAction::ConvertedToDraft {
+                    to_remove.push(Label {
+                        name: label.to_owned(),
+                    });
+                    state.data.new_pr_labels_applied = false;
```

Comment:
Hmm, not 100% sure about this one. But I guess that it makes sense; before we handled PRs opened as drafts as a special case, now being in draft is not so special anymore, and everytime you convert from draft to non-draft, it's similar to a new PR being opened 🤔

Reply:
Well we use it here: https://github.com/rust-lang/triagebot/blob/205897ccd821bf7b2de7ccc0644a90e34ed24e47/src/handlers/autolabel.rs#L103. But after this PR, we should remove that usage, to always apply the `new_pr` labels when a PR is undrafted.

Reply:
Yeah, let's do that.